spotbugs/etc/messages.xml

```diff
@@ -7025,6 +7033,29 @@ just use the constant. Methods detected are:
     ]]>
       </Details>
   </BugPattern>
+  <BugPattern type="PA_PUBLIC_ATTRIBUTES">
+    <ShortDescription>Field is both public and written by a method</ShortDescription>
+    <LongDescription>Field {1} is both public and written by a method</LongDescription>
+    <Details>
+   <![CDATA[
+    <p>
+    SEI CERT rule OBJ01-J requires that accessibility of fields must be limited:
+    https://wiki.sei.cmu.edu/confluence/display/java/OBJ01-J.+Limit+accessibility+of+fields
+    In general, requiring that no fields must be public is overkill and unrealistic. Even the rule
+    mentions that final fields may be public, except if the type of the field is a mutable reference
+    type which can be modified despite the reference itself being final. Beside final field, there
```

Comment:
Beside final field -> Besides final fields

Reply:
This typo was fixed, so this conversation can be resolved.